components/hab/src/cli.rs

```diff
@@ -607,6 +607,10 @@ pub fn get() -> App<'static, 'static> {
             )
             (subcommand: sub_svc_status())
         )
+        (@subcommand supportbundle =>
+            (about: "Create a tarball of Habitat Supervisor data to send to support")
+            (aliases: &["s", "su", "supp", "suppo", "suppor", "support-bundle"])
```

Comment:
"su" will conflict with the same alias for `sup`, so we should remove that.

I'm inclined to remove "s" too. Even though that isn't an alias for `sup`, I think most people would expect it to be, and would be confused by the creation of a support bundle.